### PR TITLE
test_: init signal schemas

### PR DESCRIPTION
**Pull request by glitchminer**
Tests of schemas for signals received during initialisation.

Important changes:
- [x] Added new reference schemas for signals during init.
- [x] ~~Changed verify_json_schema to verify_response_schema~~

Closes #

### CODE REVIEW

File: tests-functional/clients/status_backend.py [hunk 1]
```diff
@@ -61,9 +61,12 @@ def rpc_valid_request(self, method, params=[], _id=None, url=None):
         self.verify_is_valid_json_rpc_response(response, _id)
         return response
 
+    def verify_response_schema(self, response, method):
+        self.verify_json_schema(response.json(), method)
+            
     def verify_json_schema(self, response, method):
         with open(f"{option.base_dir}/schemas/{method}", "r") as schema:
```
Comment (yevh-berdnyk) [inline tests-functional/clients/status_backend.py]: I don't see any particular reason for having 2 methods that are almost the same. Let's leave only one

**Review by yevh-berdnyk — COMMENTED** [review]
Hi @glitchminer, thanks for the PR! Only one small concern from my side, otherwise looks good!

File: tests-functional/clients/status_backend.py (hunk 1 above)
Comment (glitchminer) [reply tests-functional/clients/status_backend.py]: Thanks @yevh-berdnyk. For context, the method originally only supported RPC tests. I changed it to use json directly for comparing against signals and then added `verify_response_schema` as a helper for the original RPC test methods that still pass a response object instead.

Comment (yevh-berdnyk) [reply tests-functional/clients/status_backend.py]: Yes, I see that these methods are being used in different tests. But I still suggest using only one and redoing other tests accordingly

**Review by antdanchenko — APPROVED** [review]

**Review by yevh-berdnyk — APPROVED** [review]

**Comment by igor-sirotin** [comment]
FYI, I think it should be possible to generate all JSON schemas Go code. Would we want something like that?

**Pull request merged**